cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1219,6 +1229,7 @@ pub type TxExtension = cumulus_pallet_weight_reclaim::StorageWeightReclaim<
 		frame_system::CheckWeight<Runtime>,
 		pallet_asset_conversion_tx_payment::ChargeAssetTxPayment<Runtime>,
 		frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
+		(bridge_to_rococo_config::OnAssetHubWestendRefundAssetHubRococoMessages,),
```

Comment:
```suggestion
		BridgeRejectObsoleteHeadersAndMessages,
		(bridge_to_rococo_config::OnAssetHubWestendRefundAssetHubRococoMessages,),
```

ah, I see, I also forget to add `BridgeRejectObsoleteHeadersAndMessages` to the AssetHubRococo also,

@rosarp please, fix this also for AHR, `BridgeRejectObsoleteHeadersAndMessages` is generated with macro, which you also need to add here:

```
bridge_runtime_common::generate_bridge_reject_obsolete_headers_and_messages! {
	RuntimeCall, AccountId,
	// Messages
	BridgeRococoMessages
}
```

Reply:
and also we need to fix `fn get_eth_extension` bellow, so please add these lines, then it should compile:

```
fn get_eth_extension(nonce: u32, tip: Balance) -> Self::Extension {
		(
			frame_system::CheckNonZeroSender::<Runtime>::new(),
			frame_system::CheckSpecVersion::<Runtime>::new(),
			frame_system::CheckTxVersion::<Runtime>::new(),
			frame_system::CheckGenesis::<Runtime>::new(),
			frame_system::CheckMortality::from(generic::Era::Immortal),
			frame_system::CheckNonce::<Runtime>::from(nonce),
			frame_system::CheckWeight::<Runtime>::new(),
			pallet_asset_conversion_tx_payment::ChargeAssetTxPayment::<Runtime>::from(tip, None),
			frame_metadata_hash_extension::CheckMetadataHash::<Runtime>::new(false),
+                       BridgeRejectObsoleteHeadersAndMessages, // or BridgeRejectObsoleteHeadersAndMessages::default()
+                       (
+                            bridge_to_rococo_config::OnAssetHubWestendRefundAssetHubRococoMessages::default(),
+                       ),
		)
			.into()
}
```

Reply:
here BridgeRejectObsoleteHeadersAndMessages is alias so not working. 
while BridgeRejectObsoleteHeadersAndMessages::default() does not implement Default.
Even if i add Default to its original type GenericTransactionExtensionSchema
There are other build issues. Checking it futher.

```
error[E0277]: the trait bound `GenericTransactionExtensionSchema<(), ()>: TransactionExtension<RuntimeCall>` is not satisfied
1244 |     type Extension = TxExtension;
       |                      ^^^^^^^^^^^ the trait `TransactionExtension<RuntimeCall>` is not implemented for `GenericTransactionExtensionSchema<(), ()>`, which is required by `StorageWeightReclaim<Runtime, (CheckNonZeroSender<Runtime>, CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckEra<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeAssetTxPayment<Runtime>, CheckMetadataHash<Runtime>, GenericTransactionExtensionSchema<(), ()>, (BridgeRelayersTransactionExtension<Runtime, WithMessagesExtensionConfig<StrOnAssetHubWestendRefundAssetHubRococoMessages, Runtime, Instance1, Instance1, PriorityBoostPerMessage>, HashedLaneId>,))>: TransactionExtension<RuntimeCall>`
```

Reply:
> here BridgeRejectObsoleteHeadersAndMessages is alias so not working. while BridgeRejectObsoleteHeadersAndMessages::default() does not implement Default. Even if i add Default to its original type

There is a `Default` for `BridgeRejectObsoleteHeadersAndMessages` in the macro:
```
#[derive(Clone, codec::Decode, codec::DecodeWithMemTracking, Default, codec::Encode, Eq, PartialEq, sp_runtime::RuntimeDebug, scale_info::TypeInfo)]
		pub struct BridgeRejectObsoleteHeadersAndMessages;
		impl sp_runtime::traits::TransactionExtension<$call> for BridgeRejectObsoleteHeadersAndMessages {
```

please, push adding this to the AHW:
```
bridge_runtime_common::generate_bridge_reject_obsolete_headers_and_messages! {
	RuntimeCall, AccountId,
	// Messages
	BridgeRococoMessages
}
```
and AHR:
```
bridge_runtime_common::generate_bridge_reject_obsolete_headers_and_messages! {
	RuntimeCall, AccountId,
	// Messages
	BridgeWestendMessages
}
```